plugins/@grouparoo/mailchimp/__tests__/export/export-profile.ts

```diff
@@ -570,4 +571,21 @@ describe("mailchimp/exportProfile", () => {
       })
     ).rejects.toThrow();
   });
+
+  test("can add a user passing another invalid email", async () => {
```

Comment:
Can you please clarify this test a little more - What you are testing here is specifically if mailchimp thinks the email address is invalid, grouparoo's error level will be info. 

It might also help to check that the error's message matches what mailchimp returns, e.g.: `/looks fake or invalid/`